includes/datastream.inc

```diff
@@ -138,6 +143,7 @@ function islandora_rest_datastream_post_response(array $parameters) {
   $datastream->label = isset($request['label']) ? $request['label'] : $file['name'];
   $datastream->versionable = isset($request['versionable']) ? (bool) $request['versionable'] : TRUE;
   $datastream->mimetype = isset($request['mimeType']) ? $request['mimeType'] : $mime_detect->getMimetype($file['name']);
+  $datastream->checksumType = isset($request['checksumType']) ? $request['checksumType'] : 'DISABLED';
```

Comment:
Will also add a module_exists('islandora_checksum') module, and that the current DSID is in the list defined by 'islandora_checksum_dsids_to_check'.